client/views/teams/contextualBar/info/Delete/DeleteTeamModal.js

```diff
@@ -1,30 +1,29 @@
 import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
-import React, { useState } from 'react';
+import React, { useState, useCallback } from 'react';
 
-import { StepOne, StepTwo, StepThree } from '.';
+import { StepOne, StepTwo } from '.';
 
 export const DeleteTeamModal = ({ onCancel, onConfirm, rooms }) => {
-	const [step, setStep] = useState(1);
+	const hasRooms = rooms?.length > 0;
+
+	const [step, setStep] = useState(hasRooms ? 1 : 2);
```

Comment:
We should try to maintain a certain pattern when using modal steps and I heavily recommend using this kind of pattern Instead of using just numbers because is more readable: 
```
const STEPS = {
	LIST_ROOMS: 'LIST_ROOMS',
	CONFIRM_DELETE: 'CONFIRM_DELETE',
};
```